autogen/main/cluster.tf.tmpl

```diff
@@ -992,9 +992,10 @@ resource "google_container_node_pool" "windows_pools" {
     }
 
     dynamic "advanced_machine_features" {
-      for_each = lookup(each.value, "threads_per_core", 0) > 0 ? [1] : []
+      for_each = lookup(each.value, "threads_per_core", 0) > 0 || lookup(each.value, "enable_nested_virtualization", false) ? [1] : []
       content {
-        threads_per_core = lookup(each.value, "threads_per_core", 0)
+        threads_per_core             = lookup(each.value, "threads_per_core", 0)
```

Comment:
@DrFaust92 Given this block will now be created for either condition, I think we might need this to default to null rather than 0?

```suggestion
        threads_per_core             = lookup(each.value, "threads_per_core", null)
```